components/layout_thread/lib.rs

```diff
@@ -277,7 +277,9 @@ impl LayoutThreadFactory for LayoutThread {
                     layout.start();
                 }, reporter_name, sender, Msg::CollectReports);
             }
-            let _ = content_process_shutdown_chan.send(());
+            if let Some(content_process_shutdown_chan) = content_process_shutdown_chan {
```

Comment:
It's not clear to me what the effect of this change is. Why is it ok to not have one for about:blank layout threads?